p2panda-rs/src/operation/plain/plain_value.rs

```diff
@@ -24,7 +31,10 @@ pub enum PlainValue {
     Float(f64),
 
     /// String value which can be either a text or relation (document id).
-    StringOrRelation(String),
+    String(String),
```

Comment:
Isn't this still sometimes a relation?

Reply:
Yeh, it is, but I thought it was ok to just call it `String` as this is the concrete CBOR type it represents.

Reply:
I see! But if we would go by that naming scheme then we would need to also rename ambiguous relation and relation lists because these are no cbor types? I'm fine with either, just consistent

Reply:
Cool, then I change it back cos I don't wanna change the others everywhere...